source/_docs/continuous-integration.md

```diff
@@ -17,18 +17,73 @@ Continuous Integration (CI) is a method of running automated unit and integratio
-
-After enabling the SimpleTest module, you can use Drush to remotely execute SimpleTest on your Pantheon site. For example, if you want to execute the UserSaveTestCase test and generate XML output into a writeable directory, use the following command:
-
-    SITE_NAME=yoursitename
-    ENV=dev
-    drush @pantheon.$SITE_NAME.$ENV test-run -l https://$ENV-$SITE_NAME.pantheon.io/ UserSaveTestCase --xml='sites/default/files'
-
-The end result is written to https://dev-yoursitename.pantheon.io/sites/default/files/UserSaveTestCase.xml
-
+## Drupal SimpleTest
+
+[SimpleTest](https://drupal.org/project/simpletest){.external} is a testing framework based on the [SimpleTest PHP library](https://github.com/simpletest/simpletest){.external} that is included with Drupal core. If you are creating a custom web application, you should consider including SimpleTests of your module functionality.
+
+[SiteTest](https://www.drupal.org/project/site_test){.external} is a contrib module designed to allow running tests directly against your sites code instead of a base Drupal clone of your site.  This module is recommended for use of SimpleTest on Pantheon.
+
+<div class="alert alert-info" role="alert" markdown="1">
+#### Note {.info}
+The drush test-run command was dropped in Drush 7 and 8. See [this GitHub issue](https://github.com/drush-ops/drush/issues/1362){.external} for more details.
+</div>
+
+To run tests on Pantheon:
+
+1. Enable `site_test`:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- en site_test -y
+   ```
+
+  This command will also enable `simpletest` as a dependency of `site_test`.
+
+1. Clear the cache immediately before running tests to avoid strange failures:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- cc all
+   ```
+
+  This step should be repeated each time tests are to be run moving forward.
+
+1. Get the absolute path before you can run the script:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- eval "echo DRUPAL_ROOT"
+   ```
+
+  In the command above, you may need to strip out warnings by ending the command with `2>/dev/null`.
+
+The full command will look something like this:
+
+```bash
+terminus drush $SITE_NAME.$ENV_NAME -- exec php `terminus drush $SITE_NAME.$ENV_NAME -- eval "echo DRUPAL_ROOT" 2>/dev/null`/scripts/run-tests.sh --url http://$ENV_NAME-$SITE_NAME.pantheonsite.io OptionalTestGroup
+```
+
+In the above command the `--url` option is required to be passed as multidevs do not respond to `localhost`.
+
+A full CircleCI command might look similar to this:
+
+```
+      - run:
+          name: Test simpletest
+          command: |
+            if [ "${CIRCLE_BRANCH}" != "master" ]; then
+
+              terminus drush $SITE_NAME.$ENV_NAME -- en site_test -y
```

Comment:
```suggestion
              export TERMINUS_HIDE_UPDATE_MESSAGE=1

              terminus drush $SITE_NAME.$ENV_NAME -- en site_test -y
```

We could add the export line in these two locations and it would help make this more stable.